clients/privacy-center/app/server-utils/loadEnvironmentVariables.ts

```diff
@@ -14,6 +14,10 @@ const loadEnvironmentVariables = () => {
     CONFIG_CSS_URL:
       process.env.FIDES_PRIVACY_CENTER__CONFIG_CSS_URL ||
       "file:///app/config/config.css",
+    SHOW_BRAND_LINK:
+      process.env.FIDES_PRIVACY_CENTER__SHOW_BRAND_LINK === "true" || false,
```

Comment:
Can we complicate this rollout slightly?

I'd like this to _default_ to `true`, not `false`. But I'd like to have one release to say that we're planning to do that, so we can ensure that a couple heavily-styled existing clients set this explicitly to `false` in their environment. Can you do that? Just add a tiny PR to the next release to flip this default.

Reply:
Sure-- ticketed flipping as [HJ-184](https://ethyca.atlassian.net/browse/HJ-184).